ui/imports/shared/popups/SendModal.qml

```diff
@@ -505,8 +505,8 @@ StatusDialog {
 
     footer: SendModalFooter {
         nextButtonText: popup.isBridgeTx ? qsTr("Bridge") : qsTr("Send")
-        maxFiatFees: popup.isLoading ? "..." : fees.selectedGasFiatValue
-        selectedTimeEstimate: popup.isLoading? "..." : fees.selectedTimeEstimate
+        maxFiatFees: popup.isLoading ? "..." : "%1 %2".arg(LocaleUtils.numberToLocaleString(d.totalFeesInFiat)).arg(popup.store.currentCurrency.toUpperCase())
```

Comment:
```suggestion
        maxFiatFees: popup.isLoading ? "..." : d.totalFeesInFiat.toLocaleCurrencyString(Qt.locale(), popup.store.currentCurrency))
```

Reply:
If it were a proper Number... :/

Reply:
I think you can still do this if you first convert the string to a JS Number

Reply:
Hey what I need to display is this 

![image](https://user-images.githubusercontent.com/60327365/205161887-535f4ce4-edb7-420f-9f78-b0bec8f62806.png)


seems like toLocaleCurrencyString provides a result like -` usd 123`

Reply:
@caybro can you please have a look at this too? I dont think the suggestion you made will help here.

Reply:
Ah you mean you want a different order of {currency, symbol}?

Reply:
@caybro  yes and not the symbol as in "$" but "USD"

Reply:
I see, disregard pls :)